packages/x-date-pickers/src/locales/ruRU.ts

```diff
@@ -0,0 +1,72 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+
+const ruRUPickers: PickersLocaleText<any> = {
+  // Calendar navigation
+  previousMonth: 'Предыдущий месяц',
+  nextMonth: 'Следующий месяц',
+
+  // View navigation
+  openPreviousView: 'открыть предыдущий вид',
+  openNextView: 'открыть следующий вид',
+  calendarViewSwitchingButtonAriaLabel: (view) =>
+    view === 'year'
+      ? 'открыт годовой вид, переключить на календарный вид'
+      : 'открыт календарный вид, переключить на годовой вид',
+
+  // DateRange placeholders
+  start: 'Начало',
+  end: 'Конец',
+
+  // Action bar
+  cancelButtonLabel: 'Отмена',
+  clearButtonLabel: 'Очистить',
+  okButtonLabel: 'Ок',
+  todayButtonLabel: 'Сегодня',
+
+  // Toolbar titles
+  datePickerToolbarTitle: 'Выбрать дату',
+  dateTimePickerToolbarTitle: 'Выбрать дату и время',
+  timePickerToolbarTitle: 'Выбрать время',
+  dateRangePickerToolbarTitle: 'Выбрать период',
+
+  // Clock labels
+  clockLabelText: (view, time, adapter) =>
+    `Выбрать ${view}. ${
```

Comment:
Here is the exact place I was talking about and current translation values:
<img width="1428" alt="Screenshot 2023-01-26 at 08 35 53" src="https://user-images.githubusercontent.com/4941090/214791572-331f1625-769f-44c3-81a3-1b04a37c8ad0.png">
<img width="1434" alt="Screenshot 2023-01-26 at 08 36 28" src="https://user-images.githubusercontent.com/4941090/214791604-49d0da84-78e2-4d96-8164-95ca20d46cc6.png">

And [this is ](https://github.com/mui/mui-x/blob/next/packages/x-date-pickers/src/locales/deDE.ts#L42) how fixing that part is handled in `deDE` locale. 😉

Reply:
In deDE.ts we can see 
```js
${timeViews[view] ?? view} auswählen.
```
Does the `view` can be other than `['hours', 'minutes', 'seconds']`?

Reply:
> Does the `view` can be other than `['hours', 'minutes', 'seconds']`?

Great observation. 👍 In the current state—no, those are the only valid options we'd pass.
But the mentioned snippet is possibly a bit more future-proof, once we eventually add the `milliseconds` view.
But in any case, if and when we do it—we'll have to handle all the other locales anyways.

Reply:
I noticed that the `prettier` step failed, hence you'll need to make an additional commit. If you wish, you can also add the discussed fallback at the same time.